src/working-mode

```diff
@@ -0,0 +1,242 @@
+  </ul>
+
+  <p>It’s not always possible to get the answers to all these questions in a timely manner.
+  Therefore, it might be the case that a decision is made even without input from all parties. This
+  makes it very important to file bugs for any spec changes (as discussed below), as a final check
+  to make sure all implementers are on board with, or at least aware of, the change.
+
+  <p>Additions to the standard can also be proposed as issues. However, the process for turning such
+  an issue into an accepted change to the standard is generally more involved, as the criteria for
+  additions is stricter (see Additions). A typical path for turning such issues into accepted
+  changes is detailed in the New Proposals section.
+
+  <h4>Non-normative issues</h4>
+
+  <p>Changes of editorial nature can be made, accepted, or rejected by the editor without
+  discussion.
+
+  <h3>Changes</h3>
+
+  <p>Each normative change made to the standard needs to meet the following criteria:
+
+  <ul>
+   <li>It must have support from implementers.
+   <li>It should have corresponding test changes, either in the form of new tests or modifications
+   to existing tests.
+   <li>Implementations bugs must be filed for each user agent that fails tests. (This is each user
+   agent that doesn’t match the proposed changes. If the test changes are not adequate to reveal
+   that, but it’s known through other means, the tests should be improved first.)
+   <li>It should have been reviewed by one or more members of the community.
+   <li>Optional or implementation-defined behavior must be very well motivated.
+  </ul>
+
+  <p>Additionally, the following more mechanical criteria are used to hone a change before accepting
+  it:
+
+  <ul>
+   <li>It should match the standard's source formatting and style conventions. (Consistent
+   formatting helps new contributors know what to do.)
+   <li>It must have a good commit message, with at the minimum a title, but typically with a body
+   that includes pointers to relevant discussion, tests changes, and a more elaborate description of
+   the change. (A detailed commit message helps folks in the future figure out why a decision was
+   made. This happens a lot more than you might imagine.)
+  </ul>
+
+  <p class="note">If the change is proposed by a new contributor, the editor can ensure the commit
+  message is good, and help with formatting and style. There's no need to make new contributors jump
+  through too many hoops.
+
+  <h4>Additions</h4>
+
+  <p>Any change that represents an addition needs to meet these additional criteria:
+
+  <ul>
+   <li>The addition must have the support of at least two implementers. (Such support is not binding
+   in any way on the implementers, however.)
+  </ul>
+
+  <p>Additionally, the following are strongly recommended:
+
+  <ul>
+   <li>The support from implementers should be of the form "we would like to implement this soon"
+   and not just "this seems like a reasonable idea".
```

Comment:
Didn't get to this comment before I ASCII-ified the document, but I did resolve it in a way.